### PR TITLE
JENA-1884: remove JenaUUID

### PR DESCRIPTION
**Pull request by afs**
And JENA-1885.

### CODE REVIEW

File: jena-base/src/main/java/org/apache/jena/atlas/lib/BitsInt.java [hunk 1]
```diff
@@ -155,14 +155,27 @@ boolean test(int bits, int value, int start, int finish)
...
+     * Clear the bit specified.
+     *  @param bits
+     *  @param bitIndex
+     *  @return int
+     */
+    public static final
+    int clear(int bits, int bitIndex)
+    {
+        check(bitIndex);
+        return clear$(bits, bitIndex) ;
```
Comment (kinow) [inline jena-base/src/main/java/org/apache/jena/atlas/lib/BitsInt.java:176]: `clear$` already does the `check(bitIndex)`. Would it be better to have it only there, or is it better to be safe and perform that verification on both methods?

File: jena-base/src/main/java/org/apache/jena/atlas/lib/BitsLong.java [hunk 2]
```diff
@@ -157,22 +157,35 @@ boolean test(long bits, long value, int start, int finish)
...
+    /**
+     * Clear the bit specified.
+     *  @param bits
+     *  @param bitIndex
+     *  @return long
+     */
+    public static final
+    long clear(long bits, int bitIndex)
+    {
+        check(bitIndex) ;
```
Comment (kinow) [inline jena-base/src/main/java/org/apache/jena/atlas/lib/BitsLong.java:177]: Ditto above about `check` in `clear$`

File: jena-base/src/main/java/org/apache/jena/atlas/lib/BitsInt.java (hunk 1 above)
Comment (afs) [reply jena-base/src/main/java/org/apache/jena/atlas/lib/BitsInt.java:176]: Fixed.

File: jena-base/src/main/java/org/apache/jena/atlas/lib/BitsLong.java (hunk 2 above)
Comment (afs) [reply jena-base/src/main/java/org/apache/jena/atlas/lib/BitsLong.java:177]: Can't find that one but what I have done is make the files exactly the same except s/int/long/. They now line up in diff and (easier to see!) line length.

**Pull request merged**